go/vt/srvtopo/target_stats.go

```diff
@@ -27,9 +27,6 @@ import (
 // routing of queries.
 // - discovery.TabletStatsCache will implement the discovery part of the
 //   interface, and discoverygateway will have the QueryService.
-// - hybridgateway will also implement this interface: for each l2vtgate pool,
-//   it will establish a StreamHealth connection, and store the returned
-//   health stats.
 type TargetStats interface {
```

Comment:
I believe this is also essentially obsolete. Let's look at getting rid of this at the next iteration.
Essentially, we should get rid of AggregateStats. We should only have a `map[cell]*TabletStatsCache` that vtgate directly pulls and populates from the tablets it's connected to.

Reply:
That makes sense. Sounds good.